lua/nvim-tree/view.lua

```diff
@@ -302,6 +302,21 @@ function M.open(options)
   log.profile_end(profile)
 end
 
+---@param extmarks table
+---@return number
+function M.extmarks_length(extmarks)
```

Comment:
Thanks for abstracting this.

Small request: move this to `utils.lua`. View is being converted into a stateful explorer member as part of the multi-instance view initiative: https://github.com/nvim-tree/nvim-tree.lua/pull/3109